manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,175 @@
+# Deploy code to production
+
+Transfer Digital Records uses the practice of [Continuous Integration]: developers merge their work into the a repo's
+main branch regularly. The repo is configured to only allow changes to be merged if the checks pass, such as unit tests,
+code linters and security checks. These checks are currently run on a [Jenkins instance][tdr-jenkins-intg].
+
+We also perform [Continous Delivery]. The main branch of any repo is always deployable, and can be deployed to
+production multiple times a day if necessary. The [end-to-end tests] which run on integration and staging help us catch
+regression before code is deployed to production. Deploying frequent changes to production means that each deployment is
```

Comment:
1. Typo at the start: "...developers merge their work into **the a** repo's...
2.  Shouldn't there be a colon after "such as", e.g. checks pass, such as: unit tests,...
3. Typo: "Continous" instead of "Continuous"
4. I think [Continous Delivery] was supposed to hyperlinked as it has the square brackets around it but it's not showing in the preview view
5. Should the first letter of integration, staging and production be capitalised since they are nouns?

Reply:
Fixed all of these, thanks.